rkt/image/fetcher.go

```diff
@@ -52,7 +53,7 @@ func (f *Fetcher) FetchImage(img string, ascPath string, imgType apps.AppImageTy
 			return "", err
 		}
 	}
-	if common.SupportsOverlay() {
+	if common.SupportsOverlay() && os.Geteuid() == 0 {
```

Comment:
you can comment about why we need this test, about the need to access to the tree store directories